tests/test_lemmatizer.py

```diff
@@ -452,27 +452,13 @@ def test_subwords() -> None:
 
 
 def test_is_known() -> None:
-    # logic
-    with pytest.raises(TypeError):
-        assert Lemmatizer().is_known(None, lang="en") is None  # type: ignore[arg-type]
     with pytest.raises(TypeError):
         assert is_known(None, lang="en") is None  # type: ignore[arg-type]
-    with pytest.raises(ValueError):
-        assert Lemmatizer().is_known("", lang="en") is None
     with pytest.raises(ValueError):
         assert is_known("", lang="en") is None
 
-    assert (
-        Lemmatizer().is_known("FanCY", lang="en")
-        == is_known("FanCY", lang="en")
-        == True
-    )
-    # known words
-    assert (
-        Lemmatizer().is_known("Fancy-String", lang="en")
-        == is_known("Fancy-String", lang="en")
-        == False
-    )
+    assert is_known("FanCY", lang="en") == True
+    assert is_known("Fancy-String", lang="en") == False
 
```

Comment:
Could you please add a multilingual test just to make sure everything keeps working as intended in the future?

Suggestion: `assert is_known("espejos", lang=("en", "es")) is True`